tests/test_apig_wsgi.py

```diff
@@ -472,6 +472,33 @@ def test_full_event(simple_app):
     assert simple_app.environ["apig_wsgi.full_event"] == event
 
 
+def test_elb_health_check(simple_app):
+    """Support ELB Health Check
+
+    source:
+    https://docs.aws.amazon.com/elasticloadbalancing/latest/application/lambda-functions.html#enable-health-checks-lambda  # noqa: B950
+
+    For werkzeug routing support, the get_host method should work, it relies on
+    the HTTP_HOST and wsgi.url_scheme.
+    https://github.com/pallets/werkzeug/blob/master/src/werkzeug/wsgi.py
+    """
+    event = {
+        "requestContext": {"elb": {"targetGroupArn": "..."}},
+        "httpMethod": "GET",
+        "path": "/",
+        "queryStringParameters": {},
+        "headers": {"user-agent": "ELB-HealthChecker/2.0"},
+        "body": "",
+        "isBase64Encoded": False,
+    }
```

Comment:
Just to confirm - because they're so weird as they miss this information - the events you see in practice match their documentation?

Reply:
Yes it does, and it's originated from the ALB itself, so probably doesn't have a real "http request source" hence all the missing information.